content/docs/concepts/security/index.md

```diff
@@ -365,7 +390,7 @@ targets:
    - number: 9000
 {{< /text >}}
 
-If you don't provide a `targets:` section, Istio matches the policy to all
+If you do not provide a `targets:` section, Istio matches the policy to all
```

Comment:
Why are you replacing the contractions?
Verb contractions can be ambiguous and thus should be avoided.
However, negative contractions are never ambiguous and force the reader to acknowledge the negative where as a "not" can be easily missed.
I suggest you keep the negative contractions.